### PR TITLE
All Category API Consume

### PR DESCRIPTION
**Pull request by Sodeeqishola5**
Category is showing an empty array, Take a review and feed me back.

### CODE REVIEW

File: src/pages/AllCategories.jsx [hunk 1]
```diff
@@ -16,14 +13,28 @@ const Categories = styled.section`
 `;
 
 export const AllCategories = () => {
+  const [categories, setCategories] = useState([]);
+
+  const getCategories = async () => { 
+    const response = await axios.get('https://africommerce.cyclic.app/categories');
+    setCategories(response.data.categories);
+    console.log(response.data.categories);
```
Comment (iamkelv) [inline src/pages/AllCategories.jsx]: can you remove the console.log on line 21

File: src/pages/AllCategories.jsx [hunk 2]
```diff
@@ -16,14 +13,28 @@ const Categories = styled.section`
...
+
   return (
     <Categories>
       <CategoryHeader />
-      <WomenCategory />
-      <MenCategory />
-      <TechCategory />
-      <CarCategory />
-      <KidsCategory />
+      {categories.length > 0 && categories.map((category) => (
```
Comment (iamkelv) [inline src/pages/AllCategories.jsx]: @Sodeeqishola5  can you also add a condition when the category is empty let it show "No category Found"

**Review by iamkelv — COMMENTED** [review]
@Sodeeqishola5  you can correct it

File: src/pages/AllCategories.jsx (hunk 1 above)
Comment (Sodeeqishola5) [reply src/pages/AllCategories.jsx]: OK

File: src/pages/AllCategories.jsx (hunk 2 above)
Comment (Sodeeqishola5) [reply src/pages/AllCategories.jsx]: Alright

**Pull request merged**